chat-plugins/github.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * GitHub Alert Notifications
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * This notifies a set of rooms of changes to specific repositor(y/ies).
+ *
+ * Much of this comes from Ecuacion's bot, which is based on xfix's bot,
+ * and jd who originally converted this to PS server-side.
+ *
+ * @license MIT license
+ */
+
+'use strict';
+
+let gitConfig = {};
+
+if (!Config.github) {
+	return;
```

Comment:
It'd be a better idea to use a fallback of some sort instead of returning in the module's scope like this